pyvo/io/uws/tree.py

```diff
@@ -261,7 +261,26 @@ def results(self, iterator, tag, data, config, pos):
         self._results = results
 
 
+class JobList(JobSummary, HomogeneousList):
```

Comment:
Does `JobList` got the same attributes as `JobSummary`?

Reply:
Good catch. Fixed this too. I went and fixed the `Element.parse` method to return the element - otherwise it was causing some grief.